genai-perf/genai_perf/config/input/config_analyze.py

```diff
@@ -31,11 +31,29 @@ class ConfigAnalyze(BaseConfig):
 
     def __init__(self) -> None:
         super().__init__()
+        # yapf: disable
+        sweep_parameter_template_comment = f"Uncomment the lines below to enable the analyze subcommand\n"
```

Comment:
What do you think about something like the below?

```python
sweep_parameter_template_comment = f"""
Uncomment the lines below to enable the analyze subcommand
For further details see analyze.md
sweep_parameters:
  concurrency:
    start: {AnalyzeDefaults.MIN_CONCURRENCY}
    stop: {AnalyzeDefaults.MAX_CONCURRENCY}
"""
```

Reply:
Good suggestion! I've fixed the formatting here.